README.rst

```diff
@@ -26,7 +26,7 @@ Resources
 - `Code <http://github.com/kvesteri/validators/>`_
 
 
-.. |Build Status| image:: https://travis-ci.org/kvesteri/validators.svg?branch=master
+.. |Build Status| image:: https://github.com/kvesteri/validators/workflows/tests/badge.svg
```

Comment:
@kvesteri We need you to enable GitHub Actions in the ["settings" tab](https://github.com/kvesteri/validators/settings), and then we'll know the exact badge URL to use here.

Reply:
The actions have been enabled